### ISSUE TITLE
Detailed Reports

### ISSUE THREAD

**Issue by videah** [karai17/lapis-chan#23]
At the moment reports seem very basic (I have no idea if they work at all it isn't very clear).
It would probably help if the user can describe why they are sending the report.

**Comment by karai17** [comment]
Reports do work, there is just no direct feedback. I would like to add a notification thingy to the top of the screen that gives users feedback and then disappears. Probably needs some AJAX magic in there.

I don't know if it is necessary for a user to give a detailed account of their offense. If the post in question is thought to be breaking rules, a moderator should be able to decide if that is the case without any other input.

Or maybe user input would be valuable? Perhaps preset options like "Spam" or "Illegal content" or "breaking rules" could help prioritize report investigations.

**Comment by videah** [comment]
Yeah, the idea of a small description in the report is moderators can look at a glance and prioritize reports. Ideally you would want to get rid of Illegal content first, before you get rid of shitposts, and if there is a bunch of reports in the queue then it isn't ideal.

**Comment by karai17** [comment]
Alright, I'll mark this as an enhancement and add the ability to flag a report with presets.

**Comment by shakesoda** [comment]
The idea is really just to flag posts as needing review by a mod, but I agree that there should probably be a second button indicating if something is illegal so it can be prioritized.
